hazelcast/src/main/java/com/hazelcast/jet/impl/JobRepository.java

```diff
@@ -167,28 +171,29 @@
     private static final int JOB_ID_STRING_LENGTH = idToString(0L).length();
 
 
-    private final HazelcastInstance instance;
+    private final JetInstance jetInstance;
```

Comment:
Is the `JetInstance` useful here somehow? Originally it was `HazelcastInstance`.

Reply:
I use `AbstractJetInstance#existsDistributedObject`, the logic is implemented for both client and member